<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>webItem</class>
 <widget class="QWidget" name="webItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>webItem</string>
  </property>
  <widget class="QLabel" name="image1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>350</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/Resources/dongman.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="image2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>30</y>
     <width>350</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/Resources/csdn.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>350</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>495</x>
     <y>35</y>
     <width>350</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>958</width>
     <height>258</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>image2</zorder>
  <zorder>image1</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="webItem.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>webItem</receiver>
   <slot>openWeb1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>webItem</receiver>
   <slot>openWeb2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>636</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>617</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openWeb1()</slot>
  <slot>openWeb2()</slot>
 </slots>
</ui>
